R/hooks.R

```diff
@@ -114,6 +114,7 @@ log_errors <- function(muffle = getOption('logger_muffle_errors', FALSE)) {
 #' @param level log level
 #' @param excluded_inputs character vector of input names to exclude from logging
 #' @param namespace the name of the namespace
+#' @param session passed from Shiny's \code{server}, enables to extract current Shiny namespace
```

Comment:
we can avoid adding a new parameter / have some meaningful default value if we do `session <- shiny::getDefaultReactiveDomain()` but that would mean introducing dependency on `shiny` which is probably something we would like to avoid
Alternatively we can use `` `if`(require("shiny"), getNamespace("shiny")$getDefaultReactiveDomain())`` kind of stuff. What do you think?

Reply:
Ah no. There is already `shiny::isRunning()` being used. Then I guess we can safely do `session <- shiny::getDefaultReactiveDomain()` and even avoid adding a new argument.

Reply:
didn't know about `shiny::getDefaultReactiveDomain` existence (or forgot)! If that helps to avoid `session` being an argument that we introduce and needs to be passed, then it's great. I will update the PR

Reply:
101f86984d2ab8849e7a959142948713de4638dd